### ISSUE TITLE
No rights to control the computer by espanso

### ISSUE THREAD

**Issue by jenspepie** [espanso/espanso#1539]
**Describe the bug**
Cannot use Espanso, to control the computer is allowed but the "Enable Accessibility" screen won't vanish.

**To Reproduce**
Each time on startup.

**Screenshots**
<img width="712" alt="espanso" src="https://user-images.githubusercontent.com/5345485/222475126-5893a510-faf6-4c85-939a-21aa300686a1.png">

**Logs**
16:25:28 [service(13652)] [INFO] reading configs from: "/Users/jens/Library/Application Support/espanso"
16:25:28 [service(13652)] [INFO] reading packages from: "/Users/jens/Library/Application Support/espanso/match/packages"
16:25:28 [service(13652)] [INFO] using runtime dir: "/Users/jens/Library/Caches/espanso"
16:25:28 [service(13652)] [INFO] system info: Darwin v13.2.1 - kernel: 22.3.0
16:25:28 [service(13652)] [ERROR] espanso is not running!
16:25:34 [service(13652)] [ERROR] unable to start service: timed out
16:25:34 [service(13652)] [ERROR] Hint: sometimes this happens because another Espanso process is left running for some reason.
16:25:34 [service(13652)] [ERROR]       Please try running 'espanso restart' or manually killing all Espanso processes, then try again.

**Setup information**
- OS: MacOS 13.2.1 (22D68)
- Version: 2.1.8

**Comment by AucaCoyan** [comment]
Hi! Sorry for the delay, none of the members of espanso had a mac, and I aquired one this year.
I installed espanso from brew and I'm having the same issue. I'll see why it happens

**Comment by AucaCoyan** [comment]
I have no idea, but doing `sudo tccutil reset All` and re-enabling espanso worked!
I tried with the production homebrew version, this is `espanso 2.2.1`.

That is a workaround. It shouldn't be a problem in the first place, so I'm keeping the issue open to keep a track

**Comment by mistahBen** [comment]
I wanted to add some experimental notes ( I saw some discussion on Discord but it seems more relevant here).

Fwiw, this issue seems to recur with subsequent MacOS updates. IME, I had similar issues after upgrading from 14.7 to 15.2 and then 15.3 (on the same M1 MacBook pro).

At some point, espanso stops working in a manner similar to the known secure input issue. However the secure input option in Espanso doesn't resolve the issue.

- after removing Espanso via `brew uninstall eapanso`, I also removed any lingering Espanso entries in System Settings Privacy and Accessibility options.

- I then rebooted my computer. Then installed Espanso via Homebrew. I followed the apps instructions to allow Espanso permissions, and it seemed to start working as intended.


I'm sorry I did not record any logs of the steps. 🫤

This may be hard to replicate as the issue seems to pop up *after* an OS upgrade while Espanso is installed. I also don't envy anyone digging around in Apple and MacOS's labyrinthine permissions/privilege system.

**Comment by SmileImagine** [comment]
I was experiencing this on Sequoia. The following simple sequence fixed it for me: 
Close Espanso
Settings>Privacy>Accessibility
Highlight Espanso in the list
Click the [–] button at the bottom. 
Click the [+] button at the bottom
Add Espanso
Reopen Espanso
